deploy/lib/ruler.libsonnet

```diff
@@ -0,0 +1,86 @@
+local kube = import 'kube-libsonnet/kube.libsonnet';
+
+{
+    ruler_deployment:
+        local name = $._config.ruler.name;
+        local labels = $._config.ruler.labels;
+
+        # Arguments
+        local extraArgs = $._config.ruler.extraArgs;
+        local alertmanager_uri = 'http://' + $._config.alertmanager.name + '.' + $._config.namespace + '.svc.cluster.local/api/prom/alertmanager/';
+        local configs_uri = 'http://' + $._config.configs.name + '.' + $._config.namespace + '.svc.cluster.local';
+        local consul_uri = $._config.consul.name + '.' + $._config.namespace + '.svc.cluster.local:8500';
+        local args = [
+            '-server.http-listen-port=80',
+            '-config-yaml=/etc/cortex/schemaConfig.yaml',
+            '-consul.hostname=' + consul_uri,
+            '-ruler.alertmanager-url=' + alertmanager_uri,
+            '-ruler.configs.url=' + configs_uri,
+        ];
+
+        # Environment Variables
+        local env = $._config.ruler.env;
+        local envKVMixin = $._config.ruler.envKVMixin;
+        local extraEnv = [{name: key, value: extraEnv[key]} for key in std.objectFields(envKVMixin)];
```

Comment:
should be: `local extraEnv = [{name: key, value: envKVMixin[key]} for key in std.objectFields(envKVMixin)];`